### PR TITLE
fix: Missing a piece of the URL, for the CRD installation

### LINKED ISSUE #367: Missing a piece of the URL, for the CRD installation
https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/docs/deploy-1.17-and-later.md

The version 2.0.1 is outdated:

`no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"`

When changing to 6.0.1, the url is wrong:

`server reported 404 Not Found, status code=404`

----

There seems to be an extra "client" directory added, so better update the example ?

https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd

Even bettter, to make it available as a script (.sh) to avoid all them extra `$ ` characters.

```sh
#!/bin/sh

# Change to the latest supported snapshotter version
SNAPSHOTTER_VERSION=v6.0.1

# Apply VolumeSnapshot CRDs
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml

# Create snapshot controller
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOTTER_VERSION}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml

```

### PR DESCRIPTION
**Pull request by xiangpingjiang**
**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:
fix : Missing a piece of the URL, for the CRD installation
**Which issue(s) this PR fixes**:

Fixes #367 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/ok-to-test
/lgtm

File: docs/deploy-1.17-and-later.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ error: the server doesn't have a resource type "volumesnapshotclasses"
 Next, check if any pods are running the snapshot-controller image:
 ```
 $ kubectl get pods --all-namespaces -o=jsonpath='{range .items[*]}{"\n"}{range .spec.containers[*]}{.image}{", "}{end}{end}' | grep snapshot-controller
-quay.io/k8scsi/snapshot-controller:v2.0.1, 
+quay.io/k8scsi/snapshot-controller:v6.2.1,
```
Comment (pohly) [inline docs/deploy-1.17-and-later.md:21]: There is no such image.

**Review by pohly — CHANGES_REQUESTED** [review]
/lgtm cancel

**Pull request closed without merging**